.licignore

```diff
@@ -0,0 +1,10 @@
+# Ignore everything
```

Comment:
What is the intention of this file?

Reply:
@k0da , the purpose is to filter to which files we inject license. The file works the same way as `.gitignore`.